.github/workflows/branch-lifecycle.yml

```
@@ -124,73 +124,73 @@ jobs:
 
               if [ "${{ env.BURST_MODE }}" == "true" ]; then
                 # Burst mode warning
-                cat > /tmp/pr_comment.md <<- 'EOF_BURST'
-                ⚠️ **Stale PR Warning (Burst Mode)**
+                cat > /tmp/pr_comment.md << 'EOF_BURST'
+⚠️ **Stale PR Warning (Burst Mode)**
```

actionlint (1.7.10)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)